guides/model_creation/complex_models.livemd

```diff
@@ -5,8 +5,8 @@
 ```elixir
 Mix.install([
   {:axon, github: "elixir-nx/axon"},
-  {:nx, "~> 0.3.0", github: "elixir-nx/nx", sparse: "nx", override: true},
-  {:kino, "~> 0.7.0"}
+  {:nx, github: "elixir-nx/nx", sparse: "nx", override: true},
```

Comment:
I actually think we can remove the Nx dependency from all guides, because Axon will already get the relevant one. :)